### PR TITLE
Update rbi.md

### PR DESCRIPTION
**Pull request by manhhung741**
Update the instructions to (re)generate `sorbet/rbi_list` file

### Test plan

See included automated tests.

### CODE REVIEW

File: website/docs/rbi.md [hunk 1]
```diff
@@ -181,6 +181,13 @@ the source folders for every gem in a project. If any gems have a top-level
 `rbi/` or `rbis/` folder, `srb` will collect the paths to any such folders into
 a file (`sorbet/rbi_list`) and mention this file in the `sorbet/config` file.
 
+If you add a gem that have a `rbi/` or `rbis/` folder, rerunning `sorbet-typed` 
+command will update the `sorbet/rbi_list`.
+
+```bash
+srb rbi sorbet-typed
+```
+
```
Comment (jez) [inline website/docs/rbi.md]: Is this correct for the most recent version of `srb init`? I think we removed `sorbet/rbi_list` in favor of computing + caching effectively the same list before `srb tc`.

@manhhung741 Can you check that you're on the latest version of Sorbet?

Comment (jez) [reply website/docs/rbi.md]: These changes were first introduced in `0.4.4137` but there were a number of bug fixes that didn't land until `0.4.4255`.

Comment (manhhung741) [reply website/docs/rbi.md]: Oh? I didn't know that you remove `sorbet/rbi_list`? It'd be great if you inform us these kinds of changes.

Comment (manhhung741) [reply website/docs/rbi.md]: Anyway, here is the context for this PR. Maybe you have a different answer now.

https://stackoverflow.com/questions/56741667/how-to-use-specific-rbi-file-with-tc-command/56741987#56741987

Comment (manhhung741) [reply website/docs/rbi.md]: @jez I just tested on `sorbet (0.4.4239)` and it's not working. Removing the `rbi_config` file and sorbet stops including `rbis/` from my gem

**Comment by manhhung741** [comment]
Looks like it's working with `rbi/` folder but not `rbis/`. Also working without `rbi_list`. This is now outdated. I think @ptarjan is already on it to update the documentation.

**Pull request closed without merging**